demo/client/src/ml-models/deploy-model.ts

```diff
@@ -202,10 +200,10 @@ export class ModelDeployer {
 	async deployPerceptron(model: DensePerceptronModel | SparsePerceptronModel, options: any): Promise<Contract> {
 		const { account, toFloat,
 			notify, dismissNotification,
-			saveTransactionHash, saveAddress,
+			saveTransactionHash, saveAddress, weightChunkSize = 350
```

Comment:
Getting better! I think it would be good to also keeping the names consistent for all of the methods. So would you use  `chunkSize` here?

Reply:
In the above functions there are two-parameter where I had set value, So, does I need to change over those occurrences also? if So, What would be the criteria for naming both entities

Reply:
sure will do that